changelogs/fragments/11349-add-play_name-var.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+    - Add ``play_name`` magic var (https://github.com/ansible/ansible/issues/11349)
```

Comment:
any new 'magic var' should have the `ansible_` prefix to avoid conficts

Reply:
Thanks.